routers/api/v1/repo/status.go

```diff
@@ -183,6 +219,11 @@ func GetCombinedCommitStatusByRef(ctx *context.APIContext) {
 	//   description: name of branch/tag/commit
 	//   type: string
 	//   required: true
+	// - name: page
+	//   in: query
+	//   description: page number of results
+	//   type: int
```

Comment:
IIRC this needs to be integer instead of just int
```suggestion
	//   type: integer
```